.travis.yml

```diff
@@ -75,20 +75,22 @@ jobs:
       env: SOLVER='z3' OPTS=""
 
 script:
+  # disabling Travis for now while evaluating GitHub Actions
```

Comment:
maybe consider just removing the file? if we need it later we know where to find it

Reply:
Yeah I'll definitely delete it before merging, thanks!